CHANGELOG.md

```diff
@@ -0,0 +1,245 @@
+# Changelog
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...develop)
+
+Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;".
+
+**Added:**
+
+- Indexing on TimeSeries [\#150](https://github.com/unit8co/darts/pull/150)
+- New `FourTheta` forecasting model [\#123](https://github.com/unit8co/darts/pull/123), [\#156](https://github.com/unit8co/darts/pull/156)
+- `map()` method for TimeSeries [\#121](https://github.com/unit8co/darts/issues/121), [\#166](https://github.com/unit8co/darts/pull/166)
+- Gradle to build docs, docker image, run tests, … [\#112](https://github.com/unit8co/darts/pull/112), [\#127](https://github.com/unit8co/darts/pull/127), [\#159](https://github.com/unit8co/darts/pull/159)
```

Comment:
This is not too interesting to external users - would be great to split people that use Darts to work on something from people that work on darts.

How about splitting *Added* to *New Features* and smth like *Internal*?

Reply:
Agreed that splitting changes for users and developers of the library might be useful.

I'd create new subsections ("###External" and "###Internal" maybe or better names ?) though, since each could possibly benefit from the Added / Changed / Fixed / … distinction, what do you think ?

Reply:
hmm external doesn't sound too obvious. How about ### For darts users and ### For darts developers section? I'm always bad with naming so no strong pressure on that, but would be good to make it easy to search for people that just want to use it.